pkg/network/transport/object/grpc/replication.go

```diff
@@ -206,9 +206,17 @@ func objectFromMessage(gMsg *objectGRPC.Object) (*object.Object, error) {
 }
 
 func (s *Server) metaInfoSignature(o object.Object) ([]byte, error) {
+	firstObj := o.GetFirstID()
+	if o.HasParent() && !firstObj.IsZero() {
```

Comment:
Why `!`?

Reply:
oh, sorry, fast "fix" while rebasing, it was correct before